ocaml/xapi/repository_helpers.ml

```diff
@@ -1412,18 +1413,8 @@ let get_ops_of_pending ~__context ~host ~kind =
   in
```

Comment:
For the above line:
`|> List.map (fun g -> Guidance.of_pending_guidance g)`
could directly use:
`|> List.map Guidance.of_pending_guidance`